File: prime.py
userNum = int(input("Type any number: "))
found = True # Default

for factor in range(2, userNum) :
    if int(userNum) % factor == 0 and not userNum == 2:
        found = False 
        break
if found == True and not userNum < 2 :
    print("The number is prime.")
else:
    print("The number is not prime")

File: prime.js
const findPrime = () => {
    const  num = 774;
    found = true;
    if(num < 2){
        console.log(`${num} is not a prime number`);
    }else{
        for (let i  = 2; i <= num - 1; i++)
        if (num % i == 0) {
            found = false;
            break;
        }
    if (found == true){
        console.log(`${num} is a prime number`);
    }
    else{
        console.log(`${num} is not a prime number`);
    }
}
}

findPrime();